tests/nn/pipe/skip/test_gpipe.py

```diff
@@ -24,9 +24,10 @@
 from fairscale.nn.pipe import Pipe
 from fairscale.nn.pipe.skip import pop, skippable, stash
 from fairscale.nn.pipe.skip.portal import PortalBlue, PortalCopy, PortalOrange
+from fairscale.utils.testing import skip_if_single_gpu
 
 
-@pytest.mark.skipif(not torch.cuda.is_available(), reason="cuda required")
+@skip_if_single_gpu
```

Comment:
this is not a fluke, this test actually required multiple gpus, it was just early-exiting before